src/imfilter.jl

```diff
@@ -16,7 +16,7 @@ function imfilter_gpu(img, krn)
   padkrn  = padarray(krn, Fill(zero(T), ntuple(i->0, N), padsize))
 
   # perform ifft(fft(img) .* conj.(fft(krn)))
-  fftimg = img |> CuArray |> CUFFT.fft
+  fftimg = img |> CUFFT.fft
   fftkrn = padkrn |> CuArray |> CUFFT.fft
```

Comment:
I thought that we would also move the kernel only once to the GPU as a buffer with the appropriate padding already. Did I misunderstood the idea?

Reply:
I thought this kernel optimization would be implemented in the next PR.

Reply:
Ok, let's do it in a separate PR then. I thought that this PR was about loading all arrays in the GPU, but we can do it in the next PR too.